.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -15,24 +15,34 @@ on:
 concurrency:
   # A PR number if a pull request and otherwise the commit hash. This cancels
   # queued and in-progress runs for the same PR (presubmit) or commit
-  # (postsubmit). The workflow name is prepended to avoid conflicts between
-  # different workflows.
+  # (postsubmit)
   group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
   cancel-in-progress: true
 
 jobs:
   windows_x64_msvc:
-    runs-on: windows-2022
-    defaults:
-      run:
-        shell: bash
+    runs-on: arc-runner-set
```

Comment:
Ideally we could make this workflow more agnostic to the specific runner, so it can run in forks. One code pattern for that is https://github.com/openxla/stablehlo/blob/b1c1115f070493845ad92bd4a4461f77a2aa628d/.github/workflows/buildAndTestBazel.yml#L39
```yml
    runs-on: ${{ github.repository == 'openxla/stablehlo' && 'ubuntu-22.04-64core' ||  'ubuntu-22.04'  }}
```

Something to aim for later. All the build directory setup that is specific to these Azure runners will make that difficult and will limit workflow debugging to only developers with write access to this project.

A middle ground is possible where the `runs-on` field has a branch _and_ the `Create build dir` step only runs `if ${{ github.repository == ` or `if ${{ runs-on == arc-runner-set`.